cli/src/lib/git.js

```diff
@@ -25,6 +25,19 @@ async function getGitPath() {
   }
 };
 
+export async function getDiff() {
+  const gitPath = await getGitPath();
+  try {
+    const {stdout} = await child_process.spawnP(
+      gitPath,
+      ['diff', '--name-only', 'HEAD~1'],
+    );
+    return stdout.split('\n');
+  } catch (e) {
+    return null;
```

Comment:
Would it make sense to return an empty list here and then drop the `if (diff)` in `getTestGroups`?